### PR TITLE
Claim rewards on plugins

### PR DESCRIPTION
**Pull request by tbrent**
To flag an important change that might get lost in the noise: the proposed `RewardableLibP1` implementation reverts on `claimRewards()` failure, whereas before it was try-catching the call to the wrapper. I think this is okay because of the existence of `claimRewardsSingle(..)`, but it's a change to be aware of.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+- compound-v3
+  - Emit `RewardsClaimed` event during `claimRewards()`
+- curve
+  - Make `price()` more resistant to manipulation by MEV
+  - Emit `RewardsClaimed` event during `claimRewards()`
+- convex
+  - Make `price()` more resistant to manipulation by MEV
+  - Emit `RewardsClaimed` event during `claimRewards()`
+- frax-eth
+  - Emit `RewardsClaimed` event during `claimRewards()`
```
Comment (pmckelvy1) [inline CHANGELOG.md:35]: i dont see this in this pr

Comment (tbrent) [reply CHANGELOG.md:35]: oooo I think a merge went wrong

Comment (tbrent) [reply CHANGELOG.md:35]: Nvm, there's no rewards for `frax-eth` or `yearn-v2`. Amended

File: contracts/facade/FacadeMonitor.sol [hunk 2]
```diff
@@ -157,20 +157,18 @@ contract FacadeMonitor is Initializable, OwnableUpgradeable, UUPSUpgradeable, IF
...
-                cToken = ICToken(address(cTokenVault.underlying()));
-                cTokenBal = cTokenVault.balanceOf(address(rToken.main().backingManager()));
-            } else {
-                // FLUX - Uses FToken directly (fork of CToken)
-                cToken = ICToken(address(erc20));
-                cTokenBal = cToken.balanceOf(address(rToken.main().backingManager()));
+            // (1) OLD compound-v2 uses a wrapper
+            // (2) NEW compound-v2 does not use a wrapper
+            // (3) FLUX does not use a wrapper
+            ICToken cToken = ICToken(ICToken(address(erc20)).underlying()); // case (1)
```
Comment (tbrent) [inline contracts/facade/FacadeMonitor.sol:163]: leaving handling for all 3 cases here seems correct

**Review by pmckelvy1 — APPROVED** [review]
lgtm

**Pull request merged**